lib/swtpmtest.pm

```diff
@@ -54,6 +55,8 @@ sub start_swtpm_vm {
     my $sample_xml = $guest_mode->{sample_file};
     my $guest_xml = $guest_swtpm_ver->{xml_file};
     my $swtpm_type = $guest_swtpm_ver->{version};
+    $sample_xml .= '_aarch64' if is_aarch64;
+    $sample_xml .= '.xml';
```

Comment:
I think here you could probably use `$sample_xml .= is_aarch64 ? '_aarch64' : '.xml';`